easybuild/tools/github.py

```diff
@@ -324,6 +326,14 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
         # make sure path exists, create it if necessary
         mkdir(path, parents=True)
 
+    develop = download_repo(repo=GITHUB_EASYCONFIGS_REPO, branch='develop', path=path)
+    os.chdir(develop)
```

Comment:
check whether the directory exists before you cd into it, and this should be in a `try`/`except OSError` block to cleanly report errors

also, reuse this path down below where you construct `ec_files` rather than using `GITHUB_EASYCONFIGS_REPO_DEV`?